src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Customization/Models/ResourceCredentials.cs

```diff
@@ -9,11 +9,11 @@
 
 namespace Microsoft.Azure.EventGrid.Models
 {
-    public class TopicCredentials : ServiceClientCredentials
+    public class ResourceCredentials : ServiceClientCredentials
```

Comment:
Isn't this an API breaking change?

Reply:
I just realized that now. Good catch. I did not want to have duplicated code but will need to here.